project3/styles.css

```diff
@@ -0,0 +1,373 @@
+* {
+    box-sizing: border-box;
+    transition: all 375ms linear;
+}
+
+.container {
+    background-color: hsl(228, 48%, 84%);
+    margin: 0 auto;
+    width: 375px;
+}
+
+header {
+    padding-top: 1px;
+    padding-bottom: 1px;
+    background-color: #d1d5de;
+}
+
+
+
+
+
+.logo {
+    height: 239px;
+    width: 240px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 40px;
+    margin-bottom: 20px;
+    display: block;
+    position: relative;
+}
+
+.identity {
+    text-align: left;
+    margin-top: 20px;
+}
+
+.personal {
+    margin-top: 60px;
+    text-align: left;
+}
+
+.age {
+    margin-top: 38px;
+    margin-left: 16px;
+}
+
+.category {
+    margin-top: 30px;
+    margin-bottom: 18px;
+    margin-left: 16px;
+}
+
+.field-margin {
+    margin-top: 30px;
+    margin-bottom: 18px;
+    margin-left: 16px;
+}
+
+.margins {
+    margin-left: 18px;
+    margin-right: 18px;
+}
+
+#powers {
+    margin-left: 18px;
+    margin-right: 18px;
+    height: 68px;
+    width: 291px;
+}
+
+.bulb {
+    height: 100px;
+    width: 100px;
+    margin-top: 40px;
+}
+
+
+
+.bulb {
+    height: 100px;
+    width: 100px;
+    margin-top: 40px;
+}
+
+
+.muscle {
+    height: 101px;
+    width: 100px;
+}
+
+.watch {
+    height: 100px;
+    width: 100px;
+    margin-top: 40px;
+}
+
+.caption {
+    margin-top: 19px;
+    margin-left: 150px;
+    margin-right: 50px;
+}
+
+.text-box {
+    height: 73px;
+    width: 240px;
+    margin-left: 50px;
+    margin-right: 50px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+}
+
+svg {
+    display: block;
+    margin-right: 50px;
+    margin-left: 120px;
+    width: 100%;
+
+}
+
+
+
+:root {
+    --color-red: #c61a3a;
+    --color-green: #71f79f;
+    --color-offblack: #1e2019;
+    --color-grey: #d1d5de;
+    --color-offwhite: #FFF;
+    --color-blue: #163346;
+
+}
+
+*,
+*::before *::after {
+    box-sizing: border-box;
+}
+
+
+
+h1 {
+    text-align: center;
+    font-family: 'Teko', sans-serif;
+    color: black;
+    font-size: 48px;
+    margin-top: 20px;
+    margin-bottom: 42px;
+}
+
+h2 {
+
+    margin-left: 16px;
+    font-family: 'BenchNine', sans-serif;
+
+}
+
+.great-power {
+    color: #163346;
+    margin-top: 42px;
+    margin-bottom: 57px;
+    font-size: 26px;
+    text-align: center;
+    font-family: 'BenchNine', sans-serif;
+}
+
+body {
+    background-color: var(--color-blue);
+    /* border: 10px solid var(--color-green); */
+    color: var(--color-offwhite);
+    /* font-family: var(--font-sans); */
+    font-family: 'BenchNine', sans-serif;
+    font-size: 22px;
+    position: relative;
+}
+
```

Comment:
The `body` ruleset should be one of the topmost things in your stylesheet, just after the CSS variables you declared in the `:root` ruleset. These rules set the defaults for your document, so they need to be set earlier.

Good job, though, in writing a thorough ruleset for `body` -- you included everything I'd expect to see here!